package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "treetracker-query-api",
-  "version": "1.59.0",
+  "version": "1.62.1",
```

Comment:
This version number should be automatically updated by our semantic-robot, so it shouldn't be modified in a PR, is there any difficulty for you to avoid this?

Reply:
My virtual machine that must have updated. If you can tell me please how do I do, not to climb the PR.